packages/cli/core/src/commands/ui/create/vue.spec.ts

```diff
@@ -98,13 +118,32 @@ describe('ui:create:vue', () => {
     );
   };
 
+  const doMockNodeProcess = () => {
+    jest.mock('node:process', () => ({
```

Comment:
I don't think it makes a difference to mock `cwd` at `beforeEach` since Jest will automatically hoist `jest. mock` calls to the top of the module